Makefile

```diff
@@ -15,8 +15,8 @@ GO_BUILD_RECIPE=GOOS=$(GOOS) go build -o $(BIN) $(MAIN_PACKAGE)
 build:
 	$(GO_BUILD_RECIPE)
 
+# Using "-modtime 1" to make generate target deterministic. It sets all file time stamps to unix timestamp 1
 generate: $(GOBINDATA_BIN)
-	# Using "-modtime 1" to make generate target deterministic. It sets all file time stamps to unix timestamp 1
 	go-bindata -mode 420 -modtime 1 -pkg manifests -o $(BINDATA) assets/...
```

Comment:
`420`?  share some of what you are having! ;^)

Reply:
Not all of us get to live in California! 😭